client/packages/common/src/ui/components/inputs/DatePickers/BaseDatePickerInput/BaseDatePickerInput.tsx

```diff
@@ -91,6 +91,9 @@ export const BaseDatePickerInput: FC<
             '& .MuiPickersDay-root.Mui-selected': {
               backgroundColor: `${theme.palette.secondary.main}`,
             },
+            '& .MuiPickersPopper-paper': {
+              position: 'fixed',
+            },
```

Comment:
It works but do you know why it's doing this? i.e. is it a bug in MUI?

Could you please also add a comment here about why this is needed?

Reply:
Think that's just their default popper values